spec/teaspoon/configuration_spec.rb

```diff
@@ -149,6 +152,19 @@
 
   end
 
+  describe ".coverage_ignored_files" do
+
+    it "returns an empty list if nothing was set" do
+      expect(subject.coverage_ignored_files).to eq([])
+    end
+
+    it "returns an array of  if they were comma separated" do
```

Comment:
Right, because we can't pass an array in in those cases. Do CLI flags get passed in as configuration? ie does `teaspoon --coverage_ignored_files="/spec/javascripts,/vendor/assets"` and this will automagically work?